i18n/en/docusaurus-plugin-content-docs/current/reference/units/segments.md

```diff
@@ -75,10 +75,12 @@ It can contain inside:
     |   ├── content/
     |   |     ├── sort/
     |   |     └── table/
-    |   └── index.tsx/
+    |   ├── index.tsx
+    |   └── styles.module.scss
 ```
 
 ```tsx title={layer}/{slice}/ui/index.tsx
+import { Layout } from "shared/ui";
```

Comment:
issue: I don't think it was intended for `Layout` to come from `shared`, I think it's a segment-local component, `./Layout.tsx`

Reply:
So you say that it should also appear in the [tree "Complex UI for the layer"](https://feature-sliced.design/docs/reference/units/segments#complex-ui-for-the-layer) alongside index and styles.module?

Reply:
Yes, I think that's how it was intended, but my guess is as good as yours. We may wanna find the original author of this page to see what they meant.